packages/store/index.ts

```diff
@@ -57,6 +39,6 @@ export default function createStore<State>(options: Options<State> = {}) {
       [calculateState, localState]
     )
 
-    return [localState, boundActions]
+    return [localState, store.setState.bind(store)]
```

Comment:
❓ So, this means we return the `setState` function if I'm correct, and we can use it to create specific actions in the application using it. Is that the intention?

Reply:
Yep that's correct :)